src/strategies/balance-of-with-thresholds/index.ts

```diff
@@ -0,0 +1,39 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'eleven';
```

Comment:
Hi, this should be your github username
```suggestion
export const author = 'lucid-eleven';
```